game/src/game/tutorial/learn_stick_note.rs

```
@@ -67,7 +71,7 @@ fn display_animated_example_note(
     let note_count = 3;
     let note_duration = std::cmp::max(Duration::from_secs(2), message.1.duration() / 3);
     let total_note_duration = note_duration * note_count;
-    common_context.audio_manager.play(message.1.clone());
+    let _ = common_context.audio_manager.play(message.1.clone());
```

Use `.expect("Failed to play tutorial message")` instead of ignoring it